bin/util.py

```diff
@@ -972,13 +1079,13 @@ def setlimits():
 
 
 # Subclass Popen to get rusage information.
-class ResourcePopen(subprocess.Popen):
+class ResourcePopen(subprocess.Popen[Any]):
```

Comment:
Looks like the generic type parameter `Popen` is either `str` or `bytes`, do we know which of the two? I guess not, and in that case, we could also write `str | bytes` instead of `Any` :slightly_smiling_face: